Assets/HoloToolkit-Examples/SharingWithUNET/Prefabs/Player.prefab

```diff
@@ -0,0 +1,167 @@
+%YAML 1.1
```

Comment:
Maybe rename to UnetPlayer.prefab?  There might be others who have player prefabs already made.